src/docs.js

```diff
@@ -59,11 +63,49 @@ async function documenter(options) {
   // the stream was added
   // no more entries
   tarball.finalize();
-  // tarball.pipe(process.stdout);
 
   let gzip = zlib.createGzip();
   let tgz = tarball.pipe(gzip);
 
+  if (options.publish) {
+    let auth = new client.Auth({
+      credentials: options.credentials,
+    });
+
+    let creds = await auth.awsS3Credentials('read-write', 'taskcluster-raw-docs', 'testing/');
+
+    let s3 = new aws.S3(creds.credentials);
+    let s3Stream = S3UploadStream(s3);
+
+    let upload = s3Stream.upload({
+      Bucket: 'taskcluster-raw-docs',
+      Key: 'testing/latest.tar.gz',
+    });
+
+    // handle progress
+    upload.on('part', function(details) {
+      console.log(details);
```

Comment:
We don't want to `console.log()` this all of the time. If we really want to write anything, make it a `debug()`.